src/software/localization/main_rigTransform.cpp

```diff
@@ -0,0 +1,127 @@
+#include <openMVG/rig/Rig.hpp>
+#include <openMVG/logger.hpp>
+
+#include <openMVG/sfm/sfm_data.hpp>
+#include <openMVG/sfm/sfm_data_io.hpp>
+
+#if HAVE_ALEMBIC
```

Comment:
Actually we can remove it here everywhere, because the file is added by cmake only of alembic is available